lib/absinthe/phoenix/channel.ex

```diff
@@ -52,7 +57,9 @@ defmodule Absinthe.Phoenix.Channel do
       opts,
     })
 
-    {reply, socket} = case run(query, config.schema, opts) do
+    pipeline = Map.get(config, :pipeline, {__MODULE__, :default_pipeline})
```

Comment:
Then you can just `Map.get(config, :pipeline)`

Reply:
👍  thanks! fix on the way!